sdk/keyvault/azure-security-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretClient.java

```diff
@@ -300,7 +338,10 @@ public SecretProperties updateSecretProperties(SecretProperties secretProperties
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Response<SecretProperties> updateSecretPropertiesWithResponse(SecretProperties secretProperties,
                                                                          Context context) {
-        return implClient.updateSecretPropertiesWithResponse(secretProperties, context);
+        Response<SecretBundle> response = implClient.updateSecretWithResponse(vaultUrl, secretProperties.getName(),
```

Comment:
Is there a particular reason we did  not use `callWithMappedException()` like with other `...WithResponse` methods above?

Reply:
I went through the async client and I assume it's because there's no need to map to a type other than the default `HttpResponseException`?

Reply:
Correct, there is not mapping here as the hand-written REST proxy didn't have any non-HttpResponseException mappings.